orbit-2-test/src/main/java/com/babylon/orbit2/TestStreamObserver.kt

```diff
@@ -40,9 +51,13 @@ class TestStreamObserver<T>(stream: Stream<T>) {
         }
     }
 
+    /**
+     * Closes the subscription on the underlying stream. No further values will be received after
+     * this call.
+     */
     fun close(): Unit = closeable.close()
 
-    companion object {
+    private companion object {
```

Comment:
as a general rule I think you're always better off making the individual constants private for future extensibility... with a private companion object no one else can create extension functions, right?